pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/MPSTNCuda.hpp

```diff
@@ -162,6 +168,31 @@ class MPSTNCuda final : public TNCudaBase<Precision, MPSTNCuda<Precision>> {
         setBasisState(zeroState);
     }
 
+    /**
+     * @brief Update the ith MPS site data.
+     *
+     * @param site_idx Index of the MPS site.
+     * @param host_data Pointer to the data on host.
+     * @param host_data_size Length of the data.
+     */
+    void updateIthMPSSiteData(const std::size_t site_idx,
```

Comment:
The ith can be implicit.

```suggestion
    void updateMPSSiteData(const std::size_t site_idx,
```